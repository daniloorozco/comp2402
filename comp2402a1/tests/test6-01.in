Assuming
Java
is
available
and
also
that
the
list
is
of
type
List
If
the
list
is
actually
list
of
lists
the
question
slightly
unclear
on
that
