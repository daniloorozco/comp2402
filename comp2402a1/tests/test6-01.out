If
is
of
on
and
the
Java
List
also
list
that
type
lists
unclear
Assuming
actually
question
slightly
available
